independent-projects/resteasy-reactive/client/runtime/src/main/java/org/jboss/resteasy/reactive/client/impl/ClientResponseImpl.java

```diff
@@ -67,4 +67,8 @@ restClientRequestContext.properties, restClientRequestContext, getStringHeaders(
             throw new ProcessingException(e);
         }
     }
+
+    public String getHttpVersion() {
+        return restClientRequestContext.getVertxClientResponse().version().toString();
+    }
```

Comment:
To be honest, this was convenient for testing purposes... but I do think it still makes sense as now the HTTP version can be other than HTTP 1.1.

Reply:
Not sure I follow here

Reply:
The problem is that we didn't expose any method to return the HTTP version and I needed it to verify that the HTTP version was the expected in a test. 

But in spite of it being convenient for a test, I think it makes sense to keep this method for users as well.

Reply:
I seriously doubt anyone will ever use it as it's an implementation class method.

 In any case, I'm fine with keeping it

Reply:
+1